src/dialect/postgresql.rs

```diff
@@ -34,6 +34,8 @@ impl Dialect for PostgreSqlDialect {
     fn parse_statement(&self, parser: &mut Parser) -> Option<Result<Statement, ParserError>> {
         if parser.parse_keyword(Keyword::COMMENT) {
             Some(parse_comment(parser))
+        } else if parser.parse_keyword(Keyword::VACUUM) {
```

Comment:
Can you please also add Vacuum to the "GenericDialect" which is the superset of all the other dialects, when there is no conflict. 

Also, I think it would be better to put the parsing code into `parser.rs` rather than here so it is easier to find (though I realize you are following the model of `parse_comment`